### ISSUE TITLE
508-defect-2 [AXE-CORE]: Heading levels SHOULD increase by one 

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#9170]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Heading levels **should** only increase by one. The "VA Medical Center" heading is currently h5, but are not preceded by an h3 or h4.  This should be an h3. It is currently an h5 with h4 styling.

This also happens with the following headings throughout:
- VA Medical Center
- Health care coverage 
- Contact information

> **If this is part of the forms system, please @ me in a comment, and I will assign this ticket to the VSP forms team.**

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to navigate the hierarchy of the page content using heading levels to save time and frustration.

## Environment

- Operating System: all
- Browser: all
- Screenreading device: any
- Server destination: staging & production

## Steps to Recreate

1. Enter https://staging.va.gov/caregivers/ in browser
2. Navigate through to vet-3
3. Have developer tools open, and the axe browser extension loaded
3. Run an axe audit
5. Verify that `<h5 class="vads-u-font-size--h4">VA medical center</h5>` is called out as an error of "Heading levels should only increase by one"

### Current code
```html
<h5 class="vads-u-font-size--h4">VA medical center</h5>
```

### Recommended code
```html
<h3 class="vads-u-font-size--h4">VA medical center</h3>
```

## WCAG or Vendor Guidance (optional)

* [axe-core 3.4 - Heading levels should only increase by one](https://dequeuniversity.com/rules/axe/3.4/heading-order)

## Screenshots or Trace Logs

![Screen Shot 2020-05-15 at 1 27 53 PM](https://user-images.githubusercontent.com/57469/82079456-e0507080-96b0-11ea-92ec-7a04b6980bd8.png)

![Screen Shot 2020-05-15 at 1 44 19 PM](https://user-images.githubusercontent.com/57469/82080276-3e318800-96b2-11ea-8e27-2b28f21f89f0.png)

**Comment by jenstrickland** [comment]
Validated.
